### DISCUSSION TITLE
【版本发布问题】最近几次Releases中的UserScript版，引用脚本缺失，而且原来的引用脚本版本也比较旧

### DISCUSSION THREAD

**Discussion by tqjason** [nextai-translator/nextai-translator#787] [General]
报个版本发布问题：

从v0.0.51开始的最近几次Releases中的UserScript版，引用脚本缺失，即
@require   https://cdn.jsdelivr.net/gh/yetone/openai-translator@v0.0.53/dist/userscript/index.js
index.js 不存在

而且原来的引用脚本版本也比较旧，一直是0.1.0，感觉是自动发布有点问题。

**Comment by maltoze** [comment]
https://github.com/openai-translator/openai-translator/pull/769#issuecomment-1567896453

等我有时间修复一下
